### PR TITLE
LPS-111634 Visually distinguish instance-wide and per-account roles

### PR DESCRIPTION
**Pull request by alee8888**
https://issues.liferay.com/browse/LPS-111634

@drewbrokke @pei-jung

### CODE REVIEW

File: portal-impl/src/content/Language.properties [hunk 1]
```diff
@@ -3330,6 +3330,7 @@ overline=Overline
 override-inherited-restrictions=Override Inherited Restrictions
 overwrite-the-existing-attachment-with-the-removed-one=Overwrite the existing attachment with the removed one.
 overwrite-the-existing-entry-with-the-one-from-the-recycle-bin=Overwrite the existing entry with the one from the Recycle Bin.
+owned=Owned
```
Comment (drewbrokke) [inline portal-impl/src/content/Language.properties:3333]: If it's only used in our module, let's put it there instead.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Looks great @alee8888, just the one change for the lang key.

**Review by drewbrokke — APPROVED** [review]
LGTM 👍

**Comment by drewbrokke** [comment]
ci:forward

**Comment by drewbrokke** [comment]
ci:reevaluate:1317401_66

**Comment by drewbrokke** [comment]
ci:reevaluate:1317401_66

**Comment by Sharryxue** [comment]
No specific issue found in ci:test:relevant suite.

**Comment by drewbrokke** [comment]
Thanks @Sharryxue. In that case, this pull is ready to forward.

**Comment by Sharryxue** [comment]
Cool, will manually forward this pr.

**Pull request closed without merging**